src/screens/Notifications/UpdateEmailNotificationScreen.tsx

```diff
@@ -0,0 +1,250 @@
+import { RouteProp } from '@react-navigation/native';
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { PureComponent } from 'react';
+import { Text, StyleSheet, View } from 'react-native';
+import { connect } from 'react-redux';
+
+import { Header, InputItem, ScreenTemplate, Button } from 'app/components';
+import { Route, RootStackParams, Wallet, ConfirmAddressFlowType } from 'app/consts';
+import { CreateMessage, MessageType } from 'app/helpers/MessageCreator';
+import { isEmail } from 'app/helpers/helpers';
+import { ApplicationState } from 'app/state';
+import { selectors as notificationsSelectors } from 'app/state/notifications';
+import {
+  createNotificationEmail as createNotificationEmailAction,
+  verifyNotificationEmail as verifyNotificationEmailAction,
+  checkSubscription as checkSubscriptionAction,
+  CheckSubscriptionActionCreator,
+  VerifyNotificationEmailActionCreator,
+  SetErrorActionCreator,
+  setError as setErrorAction,
+  CreateNotificationEmailActionCreator,
+  updateNotificationEmail as updateNotificationEmailAction,
+  UpdateNotificationEmailCreator,
+  updateNotificationEmailSuccess as updateNotificationEmailSuccessAction,
+  UpdateNotificationEmailSuccessAction,
+} from 'app/state/notifications/actions';
+import { selectors as walletsSelectors } from 'app/state/wallets';
+import { typography, palette } from 'app/styles';
+
+const i18n = require('../../../loc');
+
+interface Props {
+  navigation: StackNavigationProp<RootStackParams, Route.UpdateEmailNotification>;
+  route: RouteProp<RootStackParams, Route.UpdateEmailNotification>;
+  createNotificationEmail: CreateNotificationEmailActionCreator;
+  setError: SetErrorActionCreator;
+  verifyNotificationEmail: VerifyNotificationEmailActionCreator;
+  wallets: Wallet[];
+  error: string;
+  isLoading: boolean;
+  checkSubscription: CheckSubscriptionActionCreator;
+  storedEmail: string;
+  updateNotificationEmail: UpdateNotificationEmailCreator;
```

Comment:
should be `UpdateNotificationEmailActionCreator` let's keep the naming conception